app/controllers/artefacts_controller.rb

```diff
@@ -57,6 +57,7 @@ def edit
   end
 
   def create
+    @artefact.content_id = SecureRandom.uuid
```

Comment:
Worth an `||=` here, just in case?

While there's nothing that uses the API create endpoint, it's theoretically possible that it could be used and given a content_id.